iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -213,8 +221,49 @@ def validate(self, attrs):
             "new_location",
             "new_reference_instances",
         ]
-        if not any([validated_data.get(field) for field in new_fields]):
+        if not any([attrs.get(field) for field in new_fields]):
             raise serializers.ValidationError(
                 f"You must provide at least one of the following fields: {', '.join(new_fields)}."
             )
-        return validated_data
+        return attrs
+
+
+class OrgUnitChangeRequestReviewSerializer(serializers.ModelSerializer):
+    """
+    Used to approve or reject one `OrgUnitChangeRequest`.
+    """
+
+    class Meta:
+        model = OrgUnitChangeRequest
+        fields = [
+            "status",
+            "approved_fields",
+            "rejection_comment",
+        ]
+
+    def validate_status(self, value):
+        approved = OrgUnitChangeRequest.Statuses.APPROVED
```

Comment:
This totally must work, but I'm wondering if there is not a more DRFy way of doing that.

Reply:
I can't think of any other way to do this…

Reply:
Ok, no problem